linkml/generators/fhirgen.py

```diff
@@ -0,0 +1,395 @@
+"""fhirgen: FHIR-related generators
+
+Currently, only supports Enum -> ValueSet
+
+Resources
+ 1. Enum def: https://linkml.io/linkml-model/docs/EnumDefinition/
+ 2. ValueSet def: https://www.hl7.org/fhir/valueset.html
+ 3. Issue:https://github.com/linkml/linkml/issues/905
+ 4. PR: https://github.com/linkml/linkml/pull/913
+
+TODO's
+ 1. R5: Need to see if it is different than R4 for ValueSet.
+ 2. Delete all stuff from file that I don't need
+ 3. Finish mapping__enum_valueset.csv
+   - Nested objects: e.g. FHIR's `identifier`
+   - Mutation dict of func/lambda's
+   - Do all of the 1..x cardinalities in FHIR first
+ 4. Evaluate 'abritrary data model mapping' standards/formats/tools to potentially use over my ad-hoc CSV approach:
```

Comment:
@cmungall I looked through the code and examples of `linkml-transformer` (and I've looked at the presentation a few times as well), and it still feels like something is missing with respect to arbitrary data model mappings. I intend to continue going down my route, while also periodically evaluate other options further.